<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressConW</class>
 <widget class="QDialog" name="AddressConW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>地址管理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/new/prefix1/res/收货地址.png</normaloff>:/new/prefix1/res/收货地址.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,8">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font-size:18px;
font-weight:bold;
color:rgb(87,89,91);</string>
       </property>
       <property name="text">
        <string>我的收货地址:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignRight">
      <widget class="QLabel" name="labAddAddr">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline; color:#3db800;&quot;&gt;新增地址&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>所有地址</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QListWidget" name="listWidget">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>编辑</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0" alignment="Qt::AlignRight">
        <widget class="QPushButton" name="BtnSure">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="plainAddr">
         <property name="placeholderText">
          <string>请尽量填详细一点 ...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="acDel">
   <property name="text">
    <string>删除地址</string>
   </property>
  </action>
  <action name="acEdit">
   <property name="text">
    <string>编辑地址</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
